src/test/kotlin/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTServiceTest.kt

```diff
@@ -280,10 +257,10 @@ class InformatieObjectenRESTServiceTest : BehaviorSpec() {
             } returns responseRestEnkelvoudigInformatieobject
 
             When(
-                "createEnkelvoudigInformatieobjectWithUploadedFile is called by a role that is allowed to change the zaak"
+                "createEnkelvoudigInformatieobjectAndUploadFile is called by a role that is allowed to change the zaak"
```

Comment:
Probably best not to use actually method/function names in test descriptions.. I have been guilty of this as well..

Reply:
Replaced/removed method name